File: mybin/se/sysconfig/formatter/java/examples/localvar.java
class A
{
   public float calc(float c, float cx, float b)
   {
      float determinant=cx*cx*b/c;
      float n=norm(cx,b);
      SomeAlgorithm a = new SomeAlgorithm(determinant, n, b);
      return a.crunch();
   }
}

File: mybin/se/sysconfig/formatter/c/examples/localvar.cpp
   float Calc(float c, float cx, float b)
   {
      float determinant=cx*cx*b/c;
      float n=norm(cx,b);
      SomeAlgorithm *a = new SomeAlgorithm(determinant, n, b);
      SomeStruct scratch;
      return a->crunch(&scratch);
   }
